Mathlib/Geometry/Manifold/LocalInvariantProperties.lean

```diff
@@ -42,13 +42,9 @@ coincide on `s`, then `LiftPropWithinAt P g' s x` holds. We can't call it
 in the one for `LiftPropWithinAt`.
 -/
 
-
 noncomputable section
 
-open scoped Classical
-open Manifold Topology
-
-open Set Filter TopologicalSpace
+open Manifold Topology Set Filter TopologicalSpace
```

Comment:
Same complaint as on the other PR: You're probably mangling `open` and `open_locale`

Reply:
Fixed: in fact, the *first* line ought to be *open scoped*.